src/designer/elsa-workflows-studio/src/components/screens/workflow-definition-editor/elsa-workflow-test-panel/elsa-workflow-test-panel.tsx

```diff
@@ -181,7 +181,7 @@ export class ElsaWorkflowTestPanel {
       const renderEndpointUrl = () => {
         if (!message.activityData || !message.activityData["Path"]) return undefined;
 
-        const endpointUrl = this.serverUrl + '/workflows' + message.activityData["Path"].value + '?correlation=' + message.correlationId;
```

Comment:
Is this a bug that you found by accident?

Reply:
Yes, it looks like it was a bug because `message.activityData["Path"]` was not an object with `value` inside, but just a string value itself. So in the UI the endpoint url contained an `undefined` in it.

Reply:
Awesome, thanks for catching & fixing 👍🏻